### DISCUSSION TITLE
Tumblr

### DISCUSSION THREAD

**Discussion by anoraktrend** [catppuccin/catppuccin#2122] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

two ways, stylus -OR- on personal pages via [creating an official theme](https://www.tumblr.com/docs/en/theme_submission_guidelines) for tumblr blogs to select in the theme selector.

### Any additional comments?

It makes so much sense for this port to exist already, and yet it doesn't.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by esteesrfl** [comment]
stylus I think it's the best option here.

**Comment by stellophiliac** [comment]
[palettes for tumblr](https://github.com/AprilSylph/Palettes-for-Tumblr) could be used to implement these themes!

**Comment by sgoudham** [comment]
Closing as future discussion should take place in #2533
